### ISSUE TITLE
Add toggle to lock off and reset pan-zoom

### ISSUE THREAD

**Issue by StickyDigit** [jaredforrest/nonocross#57]
I can imagine a masochist wanting to pan and zoom around a huge board on a small screen. However, when it's not needed or wanted, pan-zoom often happens quite by accident, often leading to errors and swearing/shouting/sobbing.

Please add a simple in-game toggle for "pan & zoom", which when 'off' both inhibits and resets pan-zoom.

**Comment by StickyDigit** [comment]
Some follow-up observations.

I've had the spurious zoom/pan thing happen often with fingers and stylus on both phone and tablet.

I hooked my phone up to my laptop and attempted to capture the issue with "scrcpy -t -r nonocross.mkv".

Curiously, the issue refused to occur while under this form of observation. I played for ages and not once did I get a spurious zoom or pan event.

I did however catch one glitch of a kind I've seen often:-

https://user-images.githubusercontent.com/30390101/189531964-0b7837cb-6b00-4ed0-935a-121566850d7e.mp4

Near the end of the clip, an off-target touch event is seen to cause a whole chunk to be erased.

I get a lot of this sort of thing, only in nonocross. Sometimes one touch can cause seemingly random things to happen, like blocks or crosses popping up or vanishing well away from the touch event.

While running scrcpy I get very few glitches of any kind.

FWIW, the phone is a Samsung S4 mini i9195 running an unofficial LOS 18.1. The tablet is a Nexus 7 (2013) (flox), LOS (official) 18.1. Neither device has GSF. I'm using the F-Droid build of nonocross on both.

I also have a OnePkus 3T (LOS 18.1 with  GSF installed), which seems a lot less twitchy, though I haven't tested on that for very long.

**Comment by StickyDigit** [comment]
Ha! I just realised the icon bottom left resets pan/zoom. Can't imagine why I missed that given the hours I've played.

Still would like to be able to latch pan and zoom off, as when it happens by accident, it's not uncommon for some tiles to toggle, and it's easier to spot that if the whole shebang hasn't changed shape.

The spurious click behaviour persists, mostly noticeable on all my devices without GApps. FWIW, I checked for comparison and the same oddness happens in "Patterns" from Tathams Puzzles off F-Droid. Must be the only puzzle from that collection I'd not tried.

Nonocross preferred of course.
